### DISCUSSION TITLE
More languages into auto-translate option

### DISCUSSION THREAD

**Discussion by damElo850** [ReVanced/revanced-discussions#534] [Question] [answered]
More languages into auto-translate option

**Comment by dueller** [comment]
Is it so difficult to add all available languages that are shown in browser version when on pc ?

**Reply by oSumAtrIX** [reply]
Try and see

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
https://github.com/ReVanced/revanced-patches/issues/4433
